src/main/features/claim/views/defendant-company-details.njk

```diff
@@ -24,11 +24,13 @@
             ],
             addressSection = {
               heading: 'Company address',
-              hint: 'Enter the company’s registered office or a company address that has a connection with the claim.',
+              hint: 'Enter the company’s registered office or a company address that has a connection with the claim.
```

Comment:
Please do not break like that. It will require us to have matching whitespaces in translation files.